web/templates/email/user_updated.html.eex

```diff
@@ -0,0 +1,3 @@
+<div>
+  <h2><%= @name%>, your profile was updated.</h2>
```

Comment:
That would be the case if it's a popup or something, but it's an e-mail notification when someone else updates his profile, so "successfully" doesn't sound right.